starter-code/src/clue.js

```diff
@@ -1,94 +1,130 @@
-age:          26
-image:        http://www.radiotimes.com/uploads/images/Original/111967.jpg
-ocupation:   Scientist
-
-profPlum
-first_name:   Victor
-last_name:    Plum
-color:        purple
-description:  Billionare video game designer
-age:          22
-image:        https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg
-occupation:   Designer
-
-missScarlet
-first_name:   Kasandra
-last_name:    Scarlet
-color:        red
-description:  She is an A-list movie star with a dark past
-age:          31
-image:        https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg
-occupation:   Actor
-
-mrsPeacock
-first_name:   Eleanor
-last_name:    Peacock
-color:        blue
-description:  She is from a wealthy family and uses her status and money to earn popularity
-age:          36
-image:        https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-occupation:   Socialité
-
-mrMustard
-first_name:   Jack
-last_name:    Mustard
-color:        yellow
-description:  He is a former football player who tries to get by on his former glory
-age:          62
-image:        https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-occupation:   Retired Football player
+var mrGreen = {
+    first_name: "Jacob",
+    last_name: "Green",
+    color: "green",
+    description: "He has a lot of connections",
+    age: 45,
+    image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg",
+    occupation: "Entrepreneur",
+}
+
+var drOrchid = {
+    first_name: "Doctor",
+    last_name: "Orchid",
+    color: "white",
+    description: "PhD in plant toxicology.Adopted daughter of Mr.Boddy",
+    age: 26,
+    image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+    ocupation: "Scientist",
+}
+
+var profPlum = {
+    first_name: "Victor",
+    last_name: "Plum",
+    color: "purple",
+    description: "Billionare video game designer",
+    age: 22,
+    image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+    occupation: "Designer",
+}
+
+var missScarlet = {
+    first_name: "Kasandra",
+    last_name: "Scarlet",
+    color: "red",
+    description: "She is an A- list movie star with a dark past",
+    age: 31,
+    image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+    occupation: "Actor",
+}
+var mrsPeacock = {
+    first_name: "Eleanor",
+    last_name: "Peacock",
+    color: "blue",
+    description: "She is from a wealthy family and uses her status and money to earn popularity",
+    age: 36,
+    image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation: "Socialité",
+}
+
+var mrMustard = {
+    first_name: "Jack",
+    last_name: "Mustard",
+    color: "yellow",
+    description: "He is a former football player who tries to get by on his former glory",
+    age: 62,
+    image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation: "Retired Football player",
+}
```

Comment:
Perhaps it would be a bit easier to declare the objects directly in the array, as you did with rooms.
Also, there should not be a comma after the last property of an object.

e.g.: 
```
var characters = [
{ 
  first_name: "Jack",
  last_name: "Mustard",
  color: "yellow",
  description: "He is a former football player who tries to get by on his former glory",
  age: 62,
  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
  occupation: "Retired Football player"
}, 
{ 
...
},
```

Reply:
On my local file I had that already. Maybe I pushed the unfinished file I guess? I am sending it again :)